docs/troubleshooting-oauth2.md

```diff
@@ -100,13 +108,34 @@ This issue occurs when the token does not have enough scopes or permissions to a
 Follow these steps to find out which scopes or permissions are carried in the token:
 1. Open your browwser's developer tool (e.g. in Chrome or Firefox, right-click on the page and click on *Inspect* menu option)
 2. Go to the tab *Application*
-3. Select the option *Storage* > *Local Storage* in the left panel 
-4. Click on the tree option which matches the URL of the management UI 
+3. Select the option *Storage* > *Local Storage* in the left panel
+4. Click on the tree option which matches the URL of the management UI
 5. Select the Key *rabbitmq.credentials* in the right panel
-6. Copy its value 
+6. Copy its value
 7. Go to the url https://jwt.io
 8. Paste the value into the text field *Encoded*
-9. Look at the payload's text field *Decoded* 
-10. Search for the token attribute `scope` in the tokens' payload or for the value configured in `auth_oauth2.additional_scopes_key`, if any. 
-11. Once you found the appropriate token's scope attribute, find within the attribute's value any of the scopes listed above. Please, take into account if you have configured [auth_oauth2.scope_prefix](./oauth2#scope-prefix). If you did, the scopes will be named like  `myprefix_tag:administrator`. If you instead configured [scope aliases](./oauth2-examples#use-scope-aliases), you need to find the scope alias that maps to one of the scopes listed above. 
+9. Look at the payload's text field *Decoded*
+10. Search for the token attribute `scope` in the tokens' payload or for the value configured in `auth_oauth2.additional_scopes_key`, if any.
+11. Once you found the appropriate token's scope attribute, find within the attribute's value any of the scopes listed above. If [auth_oauth2.scope_prefix](./oauth2#scope-prefix) is used, it must be taken into account: the scopes will be named like  `myprefix_tag:administrator`. If [scope aliases](./oauth2-examples#use-scope-aliases) are used, find the scope alias that maps to one of the scopes listed above
```

Comment:
Change "once you found" to "When you find" - always write tocs in the present tense where possible.
Again I would change "listed above" to "previously listed"